bin/sozo/src/args.rs

```diff
@@ -70,15 +70,15 @@ impl SozoArgs {
 #[derive(Parser, Clone, Debug)]
 #[group(multiple = false)]
 pub struct ProfileSpec {
-    #[arg(short = 'P', long)]
+    #[arg(short = 'P', long, global = true)]
```

Comment:
We may not have short argument as global to avoid conflicts? Or do you think is is already in use by lot's of people like so, and we should keep it short too?

Reply:
yeah i think we should keep both because even scarb uses `P` as the short version for specifying profile